cves/CVE-2017-12616.yml

```diff
@@ -202,4 +214,9 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Not enough testing to be able to catch the unexpected behavior. The fix
+    is just to upgrade to the latest release, which since the vulnerability
+    patch requires at most a minor release upgrade seems like a reasonable
+    solution. The thing that made this vulnerability particularly bad is that
```

Comment:
This sentence reads kinds of awkwardly to me. I would put the "seems like a reasonable solution" after the word "which".